Dockerfile

```diff
@@ -14,48 +16,49 @@
 
 # Build the binary
 FROM golang:1.19 AS builder
-
 WORKDIR /workspace
+
+# Install dependencies.
+RUN apt-get update && apt-get install -y jq && mkdir bin
+
+# Install kubectl.
+RUN wget "https://dl.k8s.io/release/$(wget -q https://dl.k8s.io/release/stable.txt -O-)/bin/linux/$(uname -m | sed 's/aarch.*/arm64/;s/armv8.*/arm64/;s/x86_64/amd64/')/kubectl" -O bin/kubectl && chmod +x bin/kubectl
```

Comment:
Should we try to match the version we have in `go.mod`?

Reply:
Like the one specified in `k8s.io/kubernetes v1.24.3`? We can, although not absolutely necessary given that clients _should_ work across a number of Kubernetes versions. We can probably pass a Dockerfile arg to specify the version?